data/tcn/terms_extend.json

```diff
@@ -353,7 +353,7 @@
 	"ShipListModernLabel": "近代化改修",
 	"ShipListModernMax": "已滿",
 	"ShipListModernNonMax": "未滿",
-	"ShipListModernExtra": "特殊改修",
+	"ShipListModernExtra": "特殊改修 (提升過血量/運等數值)",
```

Comment:
this will be obviously too long to fit the width?
![image](https://github.com/user-attachments/assets/d878f732-c990-43a0-9947-ce8f31ce82c5)

Reply:
![螢幕擷取畫面 2024-07-21 191023](https://github.com/user-attachments/assets/81ccdbbb-ec73-49bc-ad14-6273a48d69ff)
now it looks like this. Tbh I didn't notice it at first because it took the extra row before I changed it on my screen in the first place